rotkehlchen/chain/ethereum/modules/uniswap/v2/common.py

```diff
@@ -295,88 +276,106 @@ def decode_uniswap_like_deposit_and_withdrawals(
             evm_inquirer=ethereum_inquirer,
             encounter=TokenEncounterInfo(tx_hash=tx_hash),
             underlying_tokens=underlying_tokens,
+            protocol=UNISWAP_PROTOCOL if token_is_uniswap_v2_lp else None,
         )
+
         if len(pool_token.underlying_tokens) == 0:
             pool_token = dataclasses.replace(pool_token, underlying_tokens=underlying_tokens)
             GlobalDBHandler().edit_evm_token(pool_token)
+            AssetResolver().clean_memory_cache(pool_token.identifier)
+
+        if pool_token.symbol in {UNISWAP_PROTOCOL, SUSHISWAP_PROTOCOL}:
+            # uniswap and sushiswap provide the same symbol for all the LP tokens. In order to
+            # provide a a better UX if the default symbol is used then change the symbol to
+            # include the symbols of the underlying tokens.
+            object.__setattr__(pool_token, 'symbol', f'{pool_token.symbol} {token0.symbol}-{token1.symbol}')  # noqa: E501
```

Comment:
perhaps it would make sense, not for this PR, but to abstract editing a token somehow out of here as it seems to involve multiple things such as editing an immutable object (LOL), editing the DB and then cleaning the cache.

Don't we do it anywhere else too?